### PR TITLE
AI-57: Fix memberships batch data entry issues

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
## Overview
_A brief description of the pull request. Try to keep it non-technical. Please mark your pull request with the appropriate [label(s)](https://github.com/civicrm/civihr/blob/staging/CONTRIBUTING.md#label-types)_

## Before
_The current status. Please provide screenshots or gifs ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)) where appropriate._

## After
_What has been changed. Please provide screenshots or gifs ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)) where appropriate._

## Technical Details
_If the PR introduces noteworthy technical changes, please describe them here. Provide code snippets if necessary_

## Comments
_Anything else you would like the reviewer to note_

---

- [ ] Tests Pass

**Before**
searchbox-issue

batch-issues

**After**

resolved-searchbox-issue

resolved-batch-issues

### CODE REVIEW

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_batch.scss [hunk 1]
```diff
@@ -146,14 +152,137 @@
...
+        border-bottom: 1px solid  $crm-background;
+        border-bottom-left-radius: $border-radius-base;
+        border-bottom-right-radius: $border-radius-base;
+
+        &.odd-row {
+          background: $crm-white;
+        }
+
+        &:last-child {
+          &:first-child {
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_batch.scss]: How come? We shouln't have last-child and first-child at the same time.

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_batch.scss [hunk 2]
```diff
@@ -146,14 +152,137 @@
...
+        border-top-left-radius: $border-radius-base;
+        border-top-right-radius: $border-radius-base;
+
+        .crm-grid-cell {
+          background: $gray-lighter;
+          border: none;
+          padding: 10px;
+          vertical-align: middle;
+
+          &:first-child {
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_batch.scss]: You shouldn't exceed 3 levels of hierarchy try to minimize this. maybe by using 

.crm-grid-header .crm-grid-cell {

or 
.crm-grid-cell:first-child

Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_batch.scss]: Fixed.

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_batch.scss (hunk 1 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_batch.scss]: Fixed.

**Pull request merged**